androidApp/main/app/src/main/java/com/simplecityapps/shuttle/ui/screens/main/MainFragment.kt

```diff
@@ -60,6 +64,9 @@ class MainFragment :
         multiSheetView = view.findViewById(R.id.multiSheetView)
 
         val navController = Navigation.findNavController(requireActivity(), R.id.navHostFragment)
+        if (navController.currentDestination == null) {
+            initializeNavGraph(navController)
```

Comment:
Our nav state is persisted after the view is destroyed (when navigating to Settings for example) so we don't to inflate a new nav graph and change the screen.